resources/datadog-agent/msi/source.wxs.erb

```diff
@@ -223,6 +233,12 @@
       <ComponentRef Id="RegisterConfVariables" />
     </Feature>
 
+    <InstallExecuteSequence>
+      <Custom Action="WixCloseApplications"
+          After="StopServices"><![CDATA[REMOVE="ALL" OR REINSTALL]]></Custom>
```

Comment:
Out of curiosity, what does that mean? Do you have a reference to build those conditions?